UPGRADE.md

```diff
@@ -22,3 +22,8 @@ application when you upgrade the package from one version to another.
     'layout' => '@layout/main.php',
 ],
 ```
+
+- Changed behavior of controller name extractor. Now it uses greedy search of namespace items with "controller(s)" 
+  postfix. For example, controller namespace is `App\AllControllers\MyController\FooBar\BazController`. Previously,
+  result was "controller/foo-bar/baz", now it is "foo-bar/baz". You can use `ViewRenderer::withControllerName()`
+  instead of `ViewRenderer::withController()` to explicitly define controller name.
```

Comment:
```suggestion
- Controller name extractor now uses greedy search of namespace items with "controller(s)" postfix. For example, for controller namespace `App\AllControllers\MyController\FooBar\BazController` previously,
  result was "controller/foo-bar/baz", now it is "foo-bar/baz". You can use `ViewRenderer::withControllerName()`
  instead of `ViewRenderer::withController()` to explicitly define controller name.
```